pgcli/pgcompleter.py

```diff
@@ -308,15 +307,33 @@ def get_column_matches(self, suggestion, word_before_cursor):
         tables = suggestion.tables
         _logger.debug("Completion column scope: %r", tables)
         scoped_cols = self.populate_scoped_cols(tables)
-
+        colit = scoped_cols.items()
+        flat_cols = []
```

Comment:
just personal preference but I find something like `flat_cols = itertools.chain(*scoped_cols.items())` a bit clearer in intent